.travis.yml

```diff
@@ -0,0 +1,40 @@
+sudo: required
+
+services:
+  - docker
+
+env:
+  SHORT_COMMIT=`echo $TRAVIS_COMMIT | cut -c1-7`
+  CURRENT_DATE=`date '+%Y%m%d%H%M%S'`
+  TAG="${CURRENT_DATE}_${SHORT_COMMIT}"
+  PATH=/snap/bin:$PATH
+
+before_install:
+  - sudo apt-get update && sudo apt-get install --only-upgrade openssl
+  - sudo pip install --user urllib3[secure] --upgrade
+  - tar -czf resources.tar.gz resources/
+  - docker build -t dnanexus/parliament2:$TAG .
+  - docker tag dnanexus/parliament2:$TAG dnanexus/parliament2
```

Comment:
What about pushing to a docker registry -- would you plan to do this manually?

Reply:
+1 to pushing here. I do it in the download agent and it makes it handy to test intermediate commits between bigger releases.

Reply:
I agree, doing this would also make it easier to test on DNAnexus automatically. A couple questions regarding this:
- Which tag do you push to? latest?
- Do you then push bigger releases manually?

Reply:
IMHO think pushing to two tags: latest AND a tag that contains commit ID and/or date works well.  Basically when I create a release then the process is simply to choose a an existing tag (either commit ID or latest, e.g.) and give it a version #/name.